source/index.md

```diff
@@ -469,134 +469,134 @@ Every cleric can choose prayers from the basic prayers and from the prayer list
 
-A character can only receive the help from one other character, unless the **GM** specifies otherwise. The helping character should roll one dice, that dice will contribute the main roll. If the helping character has a **proficiency** on that attribute they can use it to reroll that dice. The character performing the main roll may change that dice in any way.
+A character can only receive the help from one other character, unless the **GM** specifies otherwise. The helping character should roll one die; that dice will contribute to the main roll. If the helping character has a **proficiency** on that attribute, they can use it to reroll that dice. The character performing the main roll may (not?) change that dice in any way.
 
 Note that any **fatigue** generated by that dice should be received by the helping character only.
 
 Negative consequences of a failed task are also suffered by the helping character, where applicable, but not the **fatigue** rolled by the main character.
 
 ### Extra Effort
 
-If the player really wants to succeed it is possible to add one more dice to the ***available dices*** pool by causing the character one **fatigue** point before the roll, as long the player announces that **before** rolling the dice. A character cannot spend **fatigue** if it would cause the character to receive a wound as consequence of doing so.
+If the player really wants to succeed, it is possible to add one more dice to the ***available dice*** pool by causing the character one **fatigue** point before the roll, as long the player announces that **before** rolling the dice. A character cannot spend **fatigue** if it would cause the character to receive a wound as consequence of doing so.
 
-Players may use extra effort after rolling the dice.
+Players may (not?) use extra effort after rolling the dice.
 
 <pagebreak>
 
-If the player is already going for the **extra effort** rolling 1 and 4 will not cause **fatigue**, since it's already receiving that point.
+If the player is already going for the **extra effort**, rolling a 1 or a 4 will not cause **fatigue**, since they are already receiving that point.
 
 Helping characters can also use the ***extra effort*** rule, rolling two dice instead of one, as long as they announce that before rolling.
 
 ### Zero Dice?
 
-If you for any reason need to roll zero dice, you roll 2 dice instead and keep the lowest number. Rolling less than zero dice is also possible, just keep adding dice, to roll -2 dice just roll 4 dice and choose the worst. I'm pretty sure that test will fail, though, sorry.
+If for any reason you need to roll zero dice, you roll 2 dice instead and keep the lowest number. Rolling less than zero dice is also possible: add dice to ensure you have 2 total dice being rolled. For instance, to roll -2 dice when you have only 1 dice available, roll 4 dice and choose the worst. I'm pretty sure that test will fail, though, sorry.
```

Comment:
I want to make sure this is accurate.

Reply:
I think this part is a bit confusing, I'll try to reword it for more clarity.

```suggestion
If, for any reason, you need to roll zero dice, you should roll 2 die instead and take the lowest number instead. Rolling less than zero dice is also possible: roll 2 die and add one extra dice for each value below zero. For instance, when rolling -3 die, roll 5 die (2+3) and choose the worst. I'm pretty sure that test will fail, though, sorry.
```